rr_common/src/navigation_controller/navigation_controller.cpp

```diff
@@ -3,12 +3,15 @@
 #include <rr_platform/steering.h>
 #include <std_msgs/Bool.h>
 #include <std_msgs/Int8.h>
+#include <std_msgs/Empty.h>
```

Comment:
Try not to directly use primitive message types (see the [std_msgs documentation](http://wiki.ros.org/std_msgs#Overview) ). I know we break this rule a low (including in this file), but since we're adding new code with this PR, let's try to get the code headed in the right direction.
In this specific case, you can add a new (blank) *.msg file to rr_platform/msg. Give it a meaningful name (ie. RaceReset), and don't forget to add it to the CMakeLists file for that package.

The reasoning behind this is, as stated in the referenced docs above, primitive message types convey no semantic meaning (especially the empty type). In addition, using primitive message types weakens the type checking that ROS does for us when pairing up publishers and subscribers. ROS will happily connect any pub/sub pair using the Empty message type if we make a mistake. Making our own message types will only allow ROS to pair up publishers and subscribers that want to work with the right data.

Reply:
I realize that the instructions I added to the GitHub issue you're working on specified to use std_msgs/Empty. This was a mistake on my part.